### PR TITLE
Improve setup

### PR DESCRIPTION
**Pull request by joaothallis**
- Create archive folder
- Fix setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,38 +12,13 @@ Follow these steps to run bookmark locally:
...
-SAVE_SINGLEFILE = FALSE
-SAVE_READABILITY = FALSE
-SAVE_GIT = FALSE
-SAVE_MEDIA = FALSE
-SUBMIT_ARCHIVE_DOT_ORG = FALSE
-CHECK_SSL_VALIDITY = FALSE
-
-[ARCHIVE_METHOD_TOGGLES]
-SAVE_MERCURY = FALSE
-```
```
Comment (green-john) [inline README.md:39]: Not sure why this was deleted. I think we should all have the same setup, which includes having the same setup for archivebox. If we don't call this out here, archivebox will take longer to crawl sites.

File: README.md [hunk 2]
```diff
@@ -12,38 +12,13 @@ Follow these steps to run bookmark locally:
...
-
+* Setup archivebox running `./setup-archivebox.sh`
 * Make sure you have postgres running and `config/dev.exs` is setup to
 talk to your db. 
 * Install elixir dependencies with `mix deps.get`.
 * Create and migrate your database with `mix ecto.setup`.
 * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`.
 * For anonymous archives to work, you must register a new user with `anonymous@bookmark.org`
-as email.
+as email, it can be done running the seed: `run priv/repo/seeds.exs`.
```
Comment (green-john) [inline README.md:22]: I'm not sure what `run` is. Is this a mix command?

**Review by green-john — CHANGES_REQUESTED** [review]
This looks good, I am not approving because of what was removed from the README

File: README.md (hunk 1 above)
Comment (joaothallis) [reply README.md:39]: I create a script to setup archivebox to all have the same setup.

File: README.md (hunk 2 above)
Comment (joaothallis) [reply README.md:22]: It's missing the `mix`

**Review by green-john — CHANGES_REQUESTED** [review]
This looks good now. I think if you add the `mix` missing to the run command I can approve this

**Comment by joedavison** [comment]
Add the missing *mix*

File: README.md (hunk 2 above)
Comment (joedavison) [reply README.md:22]: Let's add it

**Comment by joaothallis** [comment]
I am testing locally and for some reason it's not working property

**Comment by joaothallis** [comment]
> I am testing it locally and for some reason it's not working property

That is very strange, it is only working fine if I add an `IO.inspect` to the result of `System.cmd`

**Review by joedavison — APPROVED** [review]

**Pull request merged**